### PR TITLE
Update install.md to add --cluster-version=latest

### PR DESCRIPTION
**Pull request by hamelsmu**
# Changes

added flag `--cluster-version=latest` so that people following the tutorial will not get the error that the latest version of Tekton is not compatible with whatever version of Kubernetes (for me it was v1.14 that was installed) that is installed by Default

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

cc: @jlewi

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This guide explains how to install Tekton Pipelines. It covers the following top
    ```bash
    #Example command for creating a cluster on GKE
    gcloud container clusters create $CLUSTER_NAME \
-     --zone=$CLUSTER_ZONE
+     --zone=$CLUSTER_ZONE --cluster-version=latest
```
Comment (afrittoli) [inline docs/install.md:29]: Thanks for this.

It does say version 1.15 or later on L24, but it is better to have a working command here.
NIT: `latest` here will change over time, depending on what GKE runs, so perhaps we should use `--cluster-version`. On the other side a specific version may not be available anymore at some point in time, but still I slightly  prefer a specific version.

Comment (dibyom) [reply docs/install.md:29]: yeah, I think a specific version would be nice!

Comment (hamelsmu) [reply docs/install.md:29]: ok can someone suggest a version?

Comment (afrittoli) [reply docs/install.md:29]: I would say the latest available now `1.15.11-gke.5`

**Comment by hamelsmu** [comment]
@afrittoli  thanks!  I made the changes and tested that it works 🙇

**Pull request merged**